deploy/Dockerfile

```diff
@@ -13,25 +13,34 @@ RUN export DBG="-g -Wall" && \
   make -e -C libpfm-4.10.1 && \
   make install -C libpfm-4.10.1
 
+RUN git clone https://github.com/intel/ipmctl/ && \
```

Comment:
I think that it is worth to use ipmctl branch with tag to avoid of breaking compatibility with code in cAdvisor.  I know that we've tested version mentioned [here](https://github.com/google/cadvisor/blob/master/docs/development/build.md#non-volatile-memory-support).